### ISSUE TITLE
Insert PR comments with relevant information on test failures for PHPUnit, Behat, Cypress, and Cypress-Axe.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#4941]
## Description
Cypress, Cypress-Axe, PHPUnit, and Behat output a ton of information in the course of running their tests, and the overwhelming majority of it is useless on any given run.  What _would_ be useful is to filter this information and post it to the PR thread with `github-commenter`.  I intended to do this as part of #4871 , thinking it might be as simple as passing a flag or two, but it's not -- and evaluating and implementing reporting solutions is realistically out-of-scope for the original issue, which was primarily concerned with unblocking Drupal 9.

This could conceivably be split into three (or more!) separate issues, given that the risk and effort will vary between the different affected systems.

Below, when I say "sufficiently useful," I mean the name and file of the test that failed, any error messages, backtrace if applicable, and any other information that might be useful for debugging the underlying issue.

Of course, the comment might fail to post or the information as posted in the comment might be incomplete, so it is essential that existing debugging information remain available.

## Acceptance Criteria
- [ ] A solution for more briefly (but still usefully) reporting test failures is in place for Cypress & Cypress-Axe
- [ ] A solution for more briefly (but still usefully) reporting test failures is in place for Behat
- [ ] A solution for more briefly (but still usefully) reporting test failures is in place for PHPUnit
- [ ] `va/tests/accessibility` reports a sufficiently useful comment in a PR thread iff a test fails
- [ ] `va/tests/behat` reports a sufficiently useful comment in a PR thread iff a test fails
- [ ] `va/tests/behavioral` reports a sufficiently useful comment in a PR thread iff a test fails
- [ ] `va/tests/phpunit` reports a sufficiently useful comment in a PR thread iff a test fails
- [ ] No currently available logs are made less available or unavailable.

**Comment by keisterj-oddball** [comment]
It may be of interest to run these tests from a GH action, and post the test results to s3 similar to what is currently done in content-build.
